aws/data_source_aws_resourcegroupstaggingapi_resources.go

```diff
@@ -0,0 +1,194 @@
+				MaxItems:      100,
+				Elem:          &schema.Schema{Type: schema.TypeString},
+				ConflictsWith: []string{"resource_arn_list"},
+			},
+			"filter": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 50,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"values": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							MaxItems: 20,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+					},
+				},
+			},
+			"resource_tag_mapping_list": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"resource_arn": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"compliance_details": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"compliance_status": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"keys_with_noncompliant_values": {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeString},
+									},
+									"non_compliant_keys": {
+										Type:     schema.TypeSet,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeString},
+									},
+								},
+							},
+						},
+						"tags": tagsSchemaComputed(),
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceAwsResourceGroupsTaggingAPIResourcesRead(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).resourcegroupstaggingapiconn
+
+	input := &resourcegroupstaggingapi.GetResourcesInput{}
+
+	if v, ok := d.GetOk("include_compliance_details"); ok {
+		input.IncludeComplianceDetails = aws.Bool(v.(bool))
+	}
+
+	if v, ok := d.GetOk("exclude_compliant_resources"); ok {
+		input.ExcludeCompliantResources = aws.Bool(v.(bool))
+	}
+
+	if v, ok := d.GetOk("resource_arn_list"); ok && v.(*schema.Set).Len() > 0 {
+		input.ResourceARNList = expandStringSet(v.(*schema.Set))
+	}
+
+	if v, ok := d.GetOk("filter"); ok {
+		input.TagFilters = expandAwsResourceGroupsTaggingAPITagFilters(v.([]interface{}))
+	}
+
+	if v, ok := d.GetOk("resource_type_filters"); ok && v.(*schema.Set).Len() > 0 {
+		input.ResourceTypeFilters = expandStringSet(v.(*schema.Set))
+	}
+
+	var taggings []*resourcegroupstaggingapi.ResourceTagMapping
+
+	err := conn.GetResourcesPages(input, func(page *resourcegroupstaggingapi.GetResourcesOutput, lastPage bool) bool {
+		if page == nil {
+			return !lastPage
+		}
+
+		taggings = append(taggings, page.ResourceTagMappingList...)
+		return !lastPage
+	})
+	if err != nil {
+		return err
+	}
+
+	d.SetId(meta.(*AWSClient).partition)
+
+	if err := d.Set("resource_tag_mapping_list", flattenAwsResourceGroupsTaggingAPIResourcesTagMappingList(taggings, meta)); err != nil {
```

Comment:
I think passing through provider ignore tags against this data source's result set could be confusing in this case -- if there is a requirement in the future, we can certainly add it back in.

```suggestion
	if err := d.Set("resource_tag_mapping_list", flattenAwsResourceGroupsTaggingAPIResourcesTagMappingList(taggings)); err != nil {
```